src/App.js

```diff
@@ -1,19 +1,62 @@
-import React from 'react';
-import './App.css';
+import React, { useEffect, useState } from 'react';
+import ChatLog from './components/ChatLog';
 import chatMessages from './data/messages.json';
+import './App.css';
 
 const App = () => {
+
+  const [chatEntryData, setChatEntryData] = useState([
+    {
+    id: 0,
+    sender: '',
+    body: '',
+    timeStamp: '',
+    liked: false
+    }
+  ]);
+
+  useEffect(() => {
+    setChatEntryData(chatMessages);
+  }, []);
+  
+  const [likesCount, setLikesCount] = useState(0);
```

Comment:
We should always be on the lookout for whether we need to create state or not. Creating additional pieces of state adds complexity to a project. When possible, we should prefer to calculate values instead of using state to keep track of it. 

To remove the state `likesCount` you'd need to have a method like `calculateLikesCount` that iterates over chatEntryData and adds up the likes from each message: 
```javascript
const calculateLikesCount = (chatEntryData) => {
    return chatEntryData.reduce((total, message) => {
      return message.liked ? (total += 1) : total;
    }, 0); 
};